Svg.Tests.Win/SvgClipPathTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Linq;
+using FluentAssertions;
+using NUnit.Framework;
+using SkiaSharp;
+using Svg.Interfaces;
+using Svg.Pathing;
+
+namespace Svg.Tests.Win;
+
+public class SvgClipPathTest
+{
+
+    [SetUp]
+    public void SetUp()
+    {
+        SvgPlatform.Init();
+    }
+
+    [Test]
+    public void WhenClipPathRendered_GroupElementShouldBeCLipped()
+    {
+        // Arrange
+        var pngPath = "test_clip_path.png";
+        var svgPath = "clip_path.svg";
+
+
+        var svgDoc = SvgDocument.Open<SvgDocument>("Assets\\clip_path.svg");
+
+        // Act
+        var bitMap = svgDoc.DrawDocument(backgroundColor:Color.Create(255,255,255));
+        using var file = new FileSystem().OpenWrite("Assets\\test_clip_path.png");
+        
+        bitMap.SavePng(file, 100);
+        file.Close();
+
```

Comment:
Well... as your comments are indicating: There is no **Assert** part here.
So this is technically no test.

Can you please look at the other existing tests or e.g. the planrendering tests?
There we load a bitmap of what _should_ be rendered.
Then we compare the _actually rendered_ bitmap with it and create a "diff bitmap" (i.e. a bitmap with a black dot everywhere the two bitmaps do not match)
Only if they are 90% or even 100% alike, we let the test pass!

```
using var c = TestHelper.ImageCompare(svgBitmap, pngBitmap);


            c.AssertAreSimilar(92f, svgPath);
```